### DISCUSSION TITLE
How to add extra docs sections to top nav?

### DISCUSSION THREAD

**Discussion by mikeh-35** [facebook/docusaurus#8565] [Q&A]
I want to categorise my documentation by having multiple docs sections in the top nav.

These should all behave in exactly the same way as the 'Tutorial' nav entry.

How do I do this? 

I tried adding another 'item' in docusaurus.config.js like so:

```
items: [
          {
            type: 'doc',
            docId: 'intro',
            position: 'left',
            label: 'Tutorial',
          },
          {
            type: 'doc',
            docId: 'intro2',
            position: 'left',
            label: 'Docs section 2',
          },          
          {to: '/blog', label: 'Blog', position: 'left'},
```

Where 'intro2' refers to a markdown file stored in a 'docs-section-2' subfolder

But this doesn't work.

If it is possible I'm struggling to find it anywhere in the tutorial.

**Comment by slorber** [comment]
It is hard to help you without a repro. 

"exactly the same as the 'tutorial'" is not helpful to me because I don't even know which thing you are referring to. Our own website has no tutorial tab, and the init template tutorial is using auto-generated so 🤷‍♂️ . "it doesn't work" is not helpful either, what exactly doesn't work.

If you want help, please create a repro and invest time to describe your problem properly.

Our Discord server is also a better place if you are looking for casual help.

> I want to categorise my documentation by having multiple docs sections in the top nav.

My only intuition here is that you might want to have multiple sidebars. Check our own website (Docs vs API) sources for inspiration.

**Reply by mikeh-35** [reply]
I have followed the 'fast track' instructions to get the docusaurus demo running on my localhost:3000

i.e. I have run:

`npx create-docusaurus@latest my-website classic`

Followed by:

`cd my-website`
`npx docusaurus start`

I want to add more docs sections to the top nav like this:

![image](https://user-images.githubusercontent.com/43338428/213742544-634495f0-4a21-4805-9177-1c8fd09ab784.png)

From the screenshot above you can see that the 'Tutorial' section contains individual docs pages and subfolders.

I can get a 'Docs section 2' to appear in the top nav.

But what config do I need to do in order to be able to add docs pages and subfolders to 'Docs section 2' in the same way as is done in 'Tutorial'?

I specifically want extra sections in a top nav, not in a sidebar.

**Reply by slorber** [reply]
There is no concept of "docs section".

If you want to group docs you can use either different sidebars categories, different sidebars, or different docs plugin instances.

Again my intuition is that you want a 2nd sidebar. I don't even know what you mean by "section". Show me a screenshot of a site having a "section" for you so that I can understand.

**Reply by mikeh-35** [reply]
Look at the docs for tray.io

https://tray.io/documentation/platform/getting-started/overview/

The top nav is divided into different docs sections which would be too much to put into one massive sidebar:

<img width="789" alt="image" src="https://user-images.githubusercontent.com/43338428/213762640-541ab64a-1394-44bc-a72c-8871d3643e13.png">

**Reply by slorber** [reply]
> The top nav is divided into different docs sections which would be too much to put into one massive sidebar:

Yes I see it @korylus !
Again for the 3rd time you probably want to have... multiple sidebars? 😅
(instead of "one massive sidebar" like you said)

Me:

> My only intuition here is that you might want to have multiple sidebars. Check our own website (Docs vs API) sources for inspiration.

> Again my intuition is that you want a 2nd sidebar.



---


Docs sidebar: https://docusaurus.io/docs/next
<img width="282" alt="CleanShot 2023-01-25 at 17 32 46@2x" src="https://user-images.githubusercontent.com/749374/214621004-f9c8266e-4dbe-4118-b0ef-fc1a06f2e030.png">


API sidebar: https://docusaurus.io/docs/next/cli
<img width="282" alt="CleanShot 2023-01-25 at 17 33 02@2x" src="https://user-images.githubusercontent.com/749374/214621067-106c7da4-8ad6-4eaf-b3e6-2da9757cb87f.png">



---

Another example:

https://reactnative.dev/contributing/overview

<img width="1142" alt="CleanShot 2023-01-25 at 17 39 31@2x" src="https://user-images.githubusercontent.com/749374/214622987-ae4d1be9-feb2-491f-8f63-86f444f47b7f.png">


https://reactnative.dev/community/overview

<img width="1139" alt="CleanShot 2023-01-25 at 17 39 46@2x" src="https://user-images.githubusercontent.com/749374/214623054-ae2afdf3-fc27-4df2-aef1-65964fcafda8.png">


--- 

Feature docs: https://docusaurus.io/docs/next/sidebar/multiple-sidebars

<img width="990" alt="CleanShot 2023-01-25 at 17 36 56@2x" src="https://user-images.githubusercontent.com/749374/214622153-da077730-2fe6-4299-b8e4-09f7f0a605b5.png">


I really can't help more than that 😄

**Reply by mikeh-35** [reply]
I've found a Hugo theme which does what I'm looking for.

Thanks for your help and good luck!
